src/conv-core/convcore.C

```diff
@@ -3818,7 +3818,7 @@ void ConverseCommonInit(char **argv)
   if (CmiGetArgFlagDesc(argv, "+cpd", "Used *only* in conjunction with parallel debugger"))
     {
 #if CMK_CCS_AVAILABLE
-      CpvAccess(cmiArgDebugFlag) = 1;
+      cmiArgDebugFlag = 1;
```

Comment:
This is also called on each PE. (Another potential race condition). I'll push a fix for this soon.

Reply:
Is this alright? It's basically multiple threads writing into the same variable. I see other global variables like `ccsRunning` also following the same pattern.

Reply:
I think this is a data race by technicality, but in practice it will not cause any problems.

Reply:
I was thinking that I can add an `if (CmiMyRank() == 0)` condition and then set the variable.

Reply:
Could that cause a worse race condition where another rank could get past a use of `cmiArgDebugFlag` without it having been set to 1 first?

Reply:
The right solution is to set this once before multiple threads are launched.

Reply:
Yes, that's a good idea. I've move that code that reads the variable from `ConverseCommonInit` (which runs on every PE) to `ConverseInit` (which runs on every node).